java/CMakeLists.txt

```diff
@@ -481,16 +486,23 @@ if(NOT MSVC)
   set_property(TARGET ${ROCKSDB_STATIC_LIB} PROPERTY POSITION_INDEPENDENT_CODE ON)
 endif()
 
-set(ROCKSDBJNI_STATIC_LIB rocksdbjni${ARTIFACT_SUFFIX})
-add_library(${ROCKSDBJNI_STATIC_LIB} ${JNI_NATIVE_SOURCES})
-add_dependencies(${ROCKSDBJNI_STATIC_LIB} rocksdbjni_headers)
-target_link_libraries(${ROCKSDBJNI_STATIC_LIB} ${ROCKSDB_STATIC_LIB} ${LIBS})
+if(NOT CMAKE_SYSTEM_NAME MATCHES "Android")
+  set(ROCKSDBJNI_STATIC_LIB "rocksdbjni${ARTIFACT_SUFFIX}")
```

Comment:
I don't know a great deal about Android... Is it correct to say that no one would ever use a statically compiled version of rocksdb on Android?